Makefile

```diff
@@ -26,14 +26,15 @@ help:
 	@echo "  make server          				-- start a dev server"
 	@echo "  make watch           				-- start a dev server and rebuild js and css files on changes"
 	@echo "  make background      				-- start background processes"
-	@echo "  make test            				-- run all test cases with pytest"
-	@echo "  make test-lastfailed 				-- run test that failed last"
-	@echo "  make test-clean      				-- test on new database"
+	@echo "  make test            				-- run all test cases"
+	@echo "  make pytest            			-- run all test cases with pytest"
```

Comment:
indentation of the `--` is wrong here and in the next lines

Reply:
I will still just merge as I have another Makefile PR where I fixed it and I'll rebase after!